includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -28,7 +28,12 @@ public function getData() {
 		$data = [
 			'header' => [
 				'type' => 'link-image',
+				// 'image' is deprecated, use 'img' instead
 				'image' => 'wds-company-logo-fandom-powered-by-wikia-two-lines',
+				'img' => [
```

Comment:
Where those changes applied to API tests schemas?

https://github.com/Wikia/api-tests/tree/master/apitests/DesignSystem